portal-kernel/src/com/liferay/portal/kernel/dao/orm/DefaultActionableDynamicQuery.java

```diff
@@ -385,6 +388,14 @@ protected Projection getCountProjection() {
 	}
 
 	protected int getInterval() {
+		if (_interval == 0) {
+			return GetterUtil.getInteger(
+				PropsUtil.get(
+					PropsKeys.INDEX_INTERVAL,
+					new com.liferay.portal.kernel.configuration.Filter(
+						getModelClass().getName())), Indexer.DEFAULT_INTERVAL);
+		}
+
```

Comment:
@rodrigopaulino `_interval` will never a value assigned after this change. Is this intentional?

Reply:
Our final implementation will not change `DefaultActionableDynamicQuery`, since it has many more uses in Portal beyond Search. We're just setting the interval instead: https://github.com/arboliveira/liferay-portal/pull/486

Reply:
Yeah, @lipusz. This is intentional. You see, my interpretation of the new property is that a customer might want to deliberately set it to 0 (zero). If I had decided to assign the property value to `_interval` when first entering this `if` clause, I wouldn't have the control to know if that 0 (zero) was assigned intentionally or no value was ever assigned.
Hope this clarifies.
If you think this is unnecessary and you want me to change, it's fine too. Let me know what you think. :)